mne_bids/commands/mne_bids_mark_bad_channels.py

```diff
@@ -0,0 +1,83 @@
+"""Mark channels in an existing BIDS dataset as "bad".
+
+example usage:
+$ mne_bids mark_bad_channels --ch_name="MEG 0112" --description="noisy" \
+                             --ch_name="MEG 0131" --description="flat" \
+                             --subject_id=01 --task=experiment --session=test \
```

Comment:
just a thought, should this be allowed to be a wildcard? E.g., sometimes you know that one channel is bad in all the experiments.

Reply:
You mean sth like `... --session="*"`?

Reply:
yeah, to avoid people having to write for loops in bash scripts. Don't know if it's useful or overkill.

Reply:
I would say subject_id is required and then task, session etc... can be ommitted meaning all from this subject. I would mimic the Python API of mark_bad_channels that would then do with task is None then all tasks, if session is None then all sessions etc...

Reply:
That's a great idea!